clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DeploymentResult.groovy

```diff
@@ -28,4 +27,83 @@ class DeploymentResult {
   Map <String, List<String>> deployedNamesByLocation = [:]
 
   List<Artifact> createdArtifacts = []
+  Set<Deployment> deployments = []
```

Comment:
I'd love to standardize on this method of returning data back to orca ... the previous `serverGroupNamesByRegion` and `deployedNamesByLocation` are too specific and only capture a single piece of data.